blazor/dropdown-tree/style.md

```diff
@@ -533,4 +533,50 @@ You can customize the width of the component using [Width](https://help.syncfusi
         public string PId { get; set; }
     }
 }
-```
\ No newline at end of file
+```
+
+## TextWrap
+
+Specify the boolean value to the [TextWrap](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Navigations.SfDropDownTree-2.html#Syncfusion_Blazor_Navigations_SfDropDownTree_2_TextWrap) property that indicates whether to wrap the selected items into multiple lines when the selected item’s text content exceeded the input width limit or not.
+
+```cshtml
+
+@using Syncfusion.Blazor.Navigations
+
+<SfDropDownTree @ref="tree" TItem="EmployeeData" TValue="string" Placeholder="Select an employee" Width="500px" Value="selecteditem" ShowCheckBox="true" AutoUpdateCheckState="true" Mode="DdtVisualMode.Box" TextWrap="true">
+    <DropDownTreeField TItem="EmployeeData" DataSource="Data" ID="Id" Text="Name" HasChildren="HasChild" ParentID="PId" Selected="Selected" IsChecked="IsChecked"></DropDownTreeField>
+
+</SfDropDownTree>
+
+@code {
+    SfDropDownTree<string, EmployeeData>? tree;
+    List<string> selecteditem = new() { "1" };
```

Comment:
For TextWrap you need not use Selected item and autoupdate check state

Reply:
Changes given in the following commit